task/backend/coordinator/coordinator.go

```diff
@@ -58,17 +59,21 @@ func New(logger *zap.Logger, scheduler backend.Scheduler, ts platform.TaskServic
 func (c *Coordinator) claimExistingTasks() {
 	tasks, _, err := c.TaskService.FindTasks(context.Background(), platform.TaskFilter{})
 	if err != nil {
-		c.logger.Error("failed to list tasks", zap.Error(err))
 		return
 	}
-
+	newLatestCompleted := time.Now().UTC().Format(time.RFC3339)
 	for len(tasks) > 0 {
 		for _, task := range tasks {
 			if task.Status != string(backend.TaskActive) {
 				// Don't claim inactive tasks at startup.
 				continue
 			}
 
+			task, err := c.TaskService.UpdateTask(context.Background(), task.ID, platform.TaskUpdate{LatestCompleted: &newLatestCompleted})
```

Comment:
Seems like this should happen before skipping an inactive task.

Otherwise if you activate the task after startup, it's going to catch up over the entire offline time.

Can you add a test for it?

Reply:
OO good point. Will do.

Reply:
tested and fixed here 67071a8a52c7dbff0e20f45456ef49445df48bf1